configure

```diff
@@ -1515,6 +1515,13 @@ case $_host in
 	_host_os=3ds
 	_host_cpu=arm
 	_host_alias=arm-none-eabi
+	CXX="$DEVKITPRO/devkitARM/bin/arm-none-eabi-g++"
+	AS="$DEVKITPRO/devkitARM/bin/arm-none-eabi-as"
+	AR="$DEVKITPRO/devkitARM/bin/arm-none-eabi-ar"
+	RANLIB="$DEVKITPRO/devkitARM/bin/arm-none-eabi-ranlib"
+	STRIP="$DEVKITPRO/devkitARM/bin/arm-none-eabi-strip"
+	export PKG_CONFIG_PATH=$DEVKITPRO/portlibs/3ds/lib/pkgconfig
+	export PKG_CONFIG_LIBDIR=$DEVKITPRO/portlibs/3ds/lib/pkgconfig
```

Comment:
That's not the proper way to do it.
The best way is to do like Android does: by modifying the _ar, _as, _ranlib, _strip variables